src/components/DataView/DataViewActionsRow/DataViewActionsRow.styled.ts

```diff
@@ -5,3 +5,13 @@ export const DataViewActionsRowWrapper = styled.div`
 	display: flex;
 	justify-content: space-between;
 `;
+
+export const RightControlsContainer = styled.div`
+	.custom-checkbox {
+		margin-right: 6px;
+	}
+`;
+
+export const LeftControlsContainer = styled.div`
```

Comment:
I feel the names are swapped, the LeftControlsContainer has things that render on the right, and the RightControlsContainer has things that render on the left. 

Also, please add a margin-left auto because if  devs don't pass bulk actions then all the buttons get pulled to the left.